docs/src/theme/MDXComponents.js

```diff
@@ -0,0 +1,8 @@
+import ShowForTs from '@site/src/components/ShowForTs'
+import MDXComponents from '@theme-original/MDXComponents'
+
+export default {
+  // Re-use the default mapping
+  ...MDXComponents,
+  ShowForTs: ShowForTs,
```

Comment:
```suggestion
  ShowForTs,
```

Just personal preference. Feel free to leave as-is if you want.